### PR TITLE
Vehicle page changes

### PR DESCRIPTION
**Pull request by abhishekshree**
- Updated the file structure for vehicle pages
- Changed the UI as planned (Accordions and tabs added along with model-viewer)
- data.js datastore created.
For #30 @shubhamkorde

### CODE REVIEW

**Review by shubhamkorde — COMMENTED** [review]
Make the following changes:
- The alignment of the heading on Anahita and Varun pages is not centered. It can be fixed by the textAlign center style on the div containing the heading and the subheading 
- Remove the shadow from the cad model container or make it more subtle. Something like 5px 5px 10px rgba(black, 0.2) should be fine. Adding can also be an option

Rest all is good. Good to be merged.

File: src/views/Vehicles/AnahitaPage/components/SubsystemSpecs/subs.css [hunk 1]
```diff
@@ -1,22 +1,38 @@
...
+  transition: none !important;
 }
 
-.imgCol {
-  padding-top: 20px;
-  padding-bottom: 20px;
+.spec-card:hover {
+  transition: none !important;
+  animation: none !important;
+  transform: none !important;
```
[the remark this answers is not in the document]
Comment (abhishekshree) [reply src/views/Vehicles/AnahitaPage/components/SubsystemSpecs/subs.css:18]: These tags are for removing the default card animation.

**Comment by abhishekshree** [comment]
I have updated the PR @shubhamkorde 
The changes are:
- Added correct Varun Data and images
- Removed Shadows from the cad models
- Corrected CSS classnames.

File: src/views/Vehicles/VarunPage/data.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        {
+            "name": "Operation time",
+            "spec": "4 hours of continuous operation capability and capable of underwater inspection and data collection."
+        },
+        {
+            "name": "Processor",
+            "spec": "Ordroid X2 processor with Samsung Exynyos 4412 chip and 2GB RAM"
+        },
+        {
+            "name": "Something",
```
Comment (abhishekshree) [inline src/views/Vehicles/VarunPage/data.js:22]: I didn't know what to title this...

**Review by shubhamkorde — APPROVED** [review]

**Pull request merged**